Sources/NIOHTTP2PerformanceTester/main.swift

```diff
@@ -92,6 +92,8 @@ try measureAndPrint(desc: "hpackheaders_canonical_form", benchmark: HPACKHeaderC
 try measureAndPrint(desc: "hpackheaders_canonical_form_trimming_whitespace", benchmark: HPACKHeaderCanonicalFormBenchmark(.trimmingWhitespace))
 try measureAndPrint(desc: "hpackheaders_canonical_form_trimming_whitespace_short_strings", benchmark: HPACKHeaderCanonicalFormBenchmark(.trimmingWhitespaceFromShortStrings))
 try measureAndPrint(desc: "hpackheaders_canonical_form_trimming_whitespace_long_strings", benchmark: HPACKHeaderCanonicalFormBenchmark(.trimmingWhitespaceFromLongStrings))
+try measureAndPrint(desc: "hpackheaders_normalize_httpheaders_removing_10k_conn_headers", benchmark: HPACKHeadersNormalizationOfHTTPHeadersBenchmark(httpHeaders: .manyUniqueConnectionHeaderValues(10_000), iterations: 10))
```

Comment:
Can I ask to move the initialization of these `httpHeaders` values into the `setUp` function? It's a weird thing, but it makes the allocation profilers work better in cases where we only want to see one benchmark.